onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -4076,6 +4076,16 @@ def aten_index_put(
     return result
 
 
+@torch_op("aten::_unsafe_index_put")
+def aten__unsafe_index_put(
+    self: TReal,
+    indices: Sequence[INT64],
+    values: TReal,
+    accumulate: bool = False,
+) -> TReal:
+    return aten_index_put(self, indices, values, accumulate)
```

Comment:
```suggestion
```

On line 4054 above, we can simply register the function to this name since they are the same:

```python
@torch_op(("aten::index_put", "aten::_unsafe_index_put"))
```